src/sys/k_recvfrom.c

```diff
@@ -9,13 +9,12 @@
 ssize_t k_recvfrom(int sockfd,
                    void* buf,
                    size_t len,
-                   int flags,
+                   /* int flags, */
```

Comment:
Don't forget to remove these comments and the others!

Reply:
See #200

Reply:
(in other words, we have to do this right now and we'll fix that later)